frontend/src/main/scala/bloop/engine/Interpreter.scala

```diff
@@ -129,12 +129,9 @@ object Interpreter {
       val contents = Dag.toDotGraph(state.build.dags)
       logger.info(contents)
     } else {
-      // TODO: Pretty print output of show projects, please.
       val configDirectory = state.build.origin.syntax
-      logger.info(s"Projects loaded from '$configDirectory':")
-      state.build.projects.map(_.name).sorted.foreach { projectName =>
-        logger.info(s" * $projectName")
-      }
+      logger.debug(s"Projects loaded from '$configDirectory':")
```

Comment:
https://github.com/scalacenter/bloop/blob/072b2fe908970658d112db844d0062eb4c0ee596/frontend/src/test/scala/bloop/engine/InterpreterSpec.scala#L34

This needs to be adapted

Reply:
I've already done that in my last commit.